### DISCUSSION TITLE
Does PICMI input file any possible add the AMReX parameters which not in built-in python class parameters but int c++ type input file parameters?

### DISCUSSION THREAD

**Discussion by jiecaix** [BLAST-WarpX/warpx#4432] [Q&A]
I want Write a input file in PICMI, but I find that some parameters didn't exist in python version, so I want know is any possible direct pass the amrex parameters to the output c++ version initialized input file or something like this?

**Comment by dpgrote** [comment]
Which parameters would you like to be able to set? They can be added to the PICMI interface.

**Reply by jiecaix** [reply]
Thank you for your help, the parameters i want set is "warpx.start_moving_window_step“ and "warpx.end_moving_window_step". Because the antenna laser initial method will create a laser propagate in 2 direction, so i want cut it off by setting it at the boundary. By the way, did it any way to direct inject a laser in one timestep?
